vendor/grammars/vscode-noir

```diff
@@ -0,0 +1,7 @@
+diff --git a/vendor/grammars/vscode-noir b/vendor/grammars/vscode-noir
+new file mode 160000
+index 00000000..4e0cb874
+--- /dev/null
++++ b/vendor/grammars/vscode-noir
+@@ -0,0 +1 @@
++Subproject commit 4e0cb874886c0f72fac557f7d3a8acbee134beb3
```

Comment:
Whoops. This is a file of the diff, not the submodule. You should commit the submodule change introduced when you ran the `script/add-grammar` script. We'll also need the cached license file this script generates.

Reply:
Ok I just added what was generated when the `add-grammar` script. I see that what I committed is incorrect.

I am trying to re-add it, but running into errors and I'm not sure how to fix them.

```sh
➜  linguist-fork git:(master) ✗ script/add-grammar https://github.com/noir-lang/vscode-noir
Checking Docker is installed and running
Registering new submodule: https://github.com/noir-lang/vscode-noir.git
fatal: 'vendor/grammars/vscode-noir' already exists in the index and is not a submodule
➜  linguist-fork git:(master) ✗ script/add-grammar --replace=vendor/grammars/vscode-noir https://github.com/noir-lang/vscode-noir
Checking Docker is installed and running
add-grammar: Submodule '' does not exist. Aborting.
```

I tried deleting the `vscode-noir` entry from `.gitmodules`, but no luck.

Reply:
It looks like you've committed the content of the repo and not just updated the submodule. I think you'll need to remove the directory, remove the update to `.gitmodules`, commit the changes, then run `script/add-grammar https://github.com/noir-lang/vscode-noir`

Reply:
👍 Don't forget to commit and push the license `.yml` file that should have been created at the same time.

Reply:
Hmm there wasn't one. Perhaps because of the different naming convention for LICENSE files? 

We have two licenses...
![image](https://github.com/github-linguist/linguist/assets/18372439/0ad8b483-d967-4ee2-b0df-8f746d94afb4)

Reply:
I can add this manually